src/components/UI/Input/Input.tsx

```diff
@@ -1,12 +1,29 @@
 import styled from "@emotion/styled";
-import { mixin } from "../../../styles";
+import { mixin, vars } from "../../../styles";
 
 export const InputDefault = styled.input`
   ${mixin.baseStyleInput()};
 `;
 
+interface EditableAssetTitleProps {
+  errorState?: boolean;
```

Comment:
at this point should not care about any state already. can use `hasError`

Reply:
updated